pkg/graph/schema.graphql

```diff
@@ -1398,6 +1408,9 @@ unlockAllTaskListSections(modelPlanID: UUID!): [TaskListSectionLockStatus!]!
 
 updatePlanPayments(id: UUID!, changes: PlanPaymentsChanges!): PlanPayments!
 @hasRole(role: MINT_USER)
+
+acceptNDAAgreement(accept: Boolean!): NDAAccepted!
```

Comment:
Again, probably pedantic, but since NDA stands for non-disclosure agreement, I don't think we need "Agreement" at the end here, can probably just make this `acceptNDA`

Reply:
Good call, I agree.